styles/main.css

```diff
@@ -3,30 +3,30 @@ h3, h4, h5, h6 {
     background: rgb(92,219,149);
     background: linear-gradient(90deg, rgba(92,219,149,1) 13%, rgba(55,150,69,1) 54%, rgba(255,255,255,1) 100%);
     -webkit-background-clip: text; 
-    -webkit-text-fill-color: transparent;
+    background-clip: text; 

```

Comment:
You should still use the main property in addition to vendor-prefixed properties